### PR TITLE
(New) SRS SAR Wind - create new handler - creating plot on the fly

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
@DelphiWard FYI. a new handler which creates a plot on the fly to be attached to the NetCDF.
@leonardolaiolo FYI

No need to merge it yet

**Comment by lbesnard** [comment]
The last two commits remove the dependence to Cartopy and the creation of on-the-fly plots

**Comment by lbesnard** [comment]
https://github.com/aodn/backlog/issues/4197

**Comment by lbesnard** [comment]
The last commits re-revert the changes to re-add cartopy and the creation of plots. it might happen

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,22 +15,56 @@ jobs:
...
+        sudo apt-get install -y --no-install-recommends \
+        libproj-dev \
+        libgeos-dev \
+        libffi-dev \
+        cmake \
+        libgtest-dev \
+        build-essential \
+        curl \
+        libcurl4-openssl-dev
+    - name: cartopy - build Gtest needed for proj build
```
Comment (digorgonzola) [inline .github/workflows/test.yml:39]: I think we should use a binary if possible for this so the actions don't take ages

**Review by digorgonzola — CHANGES_REQUESTED** [review]

**Pull request merged**